dashboard/app/views/home/teacher_homepage.html.haml

```diff
@@ -14,20 +14,22 @@
     }
   end
 
-  sections = current_user.sections.map do |section|
 
+  base_url = CDO.code_org_url('/teacher-dashboard#/sections/')
+
+  sections = current_user.sections.map do |section|
```

Comment:
I realize this is moving existing code, but it would be good if this could be pushed down to the controller or model.  E.g. `Script#summarize`.  Doesn't need to block merging this PR.